Source/IGListKit/IGListSectionController.h

```diff
@@ -111,6 +111,18 @@ NS_SWIFT_NAME(ListSectionController)
  */
 - (void)didUnhighlightItemAtIndex:(NSInteger)index;
 
+/**
+ Tells the section controller that the cell has requested a menu configuration.
+ 
+ @param index The index of the cell that requested the menu.
+ @param point The point of the tap on the cell.
+ 
+ @return An object that conforms to `UIContextMenuConfiguration`
+ 
+ @note The default implementation does nothing. **Calling super is not required.**
+ */
+- (nullable UIContextMenuConfiguration *)contextMenuConfigurationForItemAtIndex:(NSInteger)index point:(CGPoint)point API_AVAILABLE(ios(13.0)) API_UNAVAILABLE(tvos);
```

Comment:
Does this actually compile or work? I saw you have some lint error here: https://travis-ci.org/Instagram/IGListKit/jobs/655541844?utm_medium=notification&utm_source=github_status which is related to this file change.

Reply:
Yes it does, I don't really get why it complains here, it seems it wants the nullable to be apart from the return type.
I double checked on all samples, deintegrating / reinstalling all pods, it works pretty well, on multiple versions.

Reply:
Here's the local execution of the lint: 

```
$ machine:IGListKit jbourdev$ bundle exec pod lib lint IGListSwiftKit.podspec --allow-warnings "--include-podspecs=*.podspec"

 -> IGListSwiftKit (4.1.0)
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Planning build
    - NOTE  | xcodebuild:  note: Constructing build description
    - NOTE  | xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')

IGListSwiftKit passed validation.
```

Reply:
Travis builds on Xcode 10.2.1 - 10E1001 while I locally build on Xcode 11.3.1 - 11C504. It shouldn't be an issue though.

Reply:
Same result if I do my setup the same as the CI... @lorixx any idea? :/

Reply:
@lorixx The error is different from a CI run to another... It compiles properly locally, on all targets, and for each example app. How could I solve the issue so this PR gets accepted?